packages/cli/test/unit/commands/project/list.test.ts

```diff
@@ -129,7 +129,7 @@ describe('list', () => {
       ...defaultProject,
       targets: {
         production: {
-          ...defaultProject!.targets!.production,
+          ...defaultProject!.targets!.production!,
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** remove non-null assertion

Can we explicitly check this value and throw a clear message if it's not defined?‏